r_gl.h

```diff
@@ -49,8 +49,8 @@ class GLRender : public IRender {
 	void ExecuteDrawCmds(std::vector<GLBatchDrawCmd>& drawCmds, GeometryType geomType);
 	void InitShaders();
 	void RegisterColliderModels();
-	GLBatchDrawCmd AddLineToBatch(GLBatch* btach, Vertex* verts, uint32_t numVerts, bool close);
-	GLBatchDrawCmd AddTrisToBatch(GLBatch* batch, Tri* tris, uint32_t numTris, bool cullFace, DrawMode drawMode);
+	static GLBatchDrawCmd AddLineToBatch(GLBatch* btach, Vertex* verts, uint32_t numVerts, bool close);
+	static GLBatchDrawCmd AddTrisToBatch(GLBatch* batch, Tri* tris, uint32_t numTris, bool cullFace, DrawMode drawMode);
 
```

Comment:
this is probably not right, is it?

Reply:
what do you mean? ```static``` ?

Reply:
This was in r_gl.h? How did that get in there??